### PR TITLE
chore(deps): update npm dependencies to latest versions

### PR DESCRIPTION
**Pull request by oliverdietsche**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,28 +55,28 @@
...
+    "npm-check-updates": "11.5.13",
     "npm-run-all": "4.1.5",
-    "prettier": "2.2.1",
-    "typescript": "4.1.3"
+    "prettier": "2.3.0",
+    "typescript": "4.2.4"
   },
   "peerDependencies": {
-    "eslint": ">=6",
+    "eslint": ">=7",
```
Comment (ernscht) [inline package.json:70]: wouldn't change this, if not really needed

**Review by ernscht — COMMENTED** [review]
For dependency updates I would always reinstall the whole package. Use "npm run clean" before installing.

**Review by danielkoch — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (danielkoch) [reply package.json]: Downgraded.

[the remark this answers is not in the document]
Comment (danielkoch) [reply package.json]: See comment above.

**Comment by danielkoch** [comment]
> For dependency updates I would always reinstall the whole package. Use "npm run clean" before installing.

Done.

**Pull request merged**